perfzero/lib/perfzero/utils.py

```diff
@@ -343,6 +343,68 @@ def get_gpu_info():
   return gpu_info
 
 
+def install_tpu_tool():
```

Comment:
this should be private?

Reply:
Yes, good point. It was called in benchmark.py previously, but now only needed in setup_tpu() and cleanup_tpu() within this module.